setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --reuse-db --instafail --showlocals
+addopts =-vvs --reuse-db --instafail --showlocals --tb=short
```

Comment:
I would change that in a different PR. I don't really mind, but others might, and more importantly I feel there are other changes we should make to the pytest output (with faster builds, I believe we should revert #1545 and let failures be shown at the bottom - see https://github.com/mozilla/addons-server/issues/1497)